### ISSUE TITLE
research question for spring 2020

### ISSUE THREAD

**Issue by lmcneil7** [lmcneil7/teen-titans#23]
As I mentioned in my project proposal, I wasn't too sure what to choose for this semester and thought it would be a good idea to open it to the other members. I have a couple of ideas already, but please let me know if you have any ideas!

P.S. I don't think I mention it in any of the ideas, but season 2 is based on **The Judas Contract**, issues 42-44 of The New Teen Titans. I didn't do it last semester, but it's short, and the comics we marked up are the build-up of Terra's character, so it's still relevant. 

**WARNING: I'm expanding on each idea to explain them to the best ability that I can, which means a lot of reading! Sorry in advance!**

_My ideas:_

- If you read my project proposal for _Adventure Time_, this first idea is similar to that one, but on a smaller scale. The idea is to create a virtual map of Jump City, the location of the tv series. We already did some mark-up on setting through the tag ```stagDesc``` and can focus on the area more as we mark-up season 2. This season is focused on the introduction and breakdown of Terra and based on the comic "The Judas Contract." One problem I see with this is the creation of the map. One way is to look for someone who has already created a diagram. Another way is to establish a vague outline mostly to show the locations instead of being detailed. The main focus could be comparing the comics and the tv series in where they differentiate via maps and fight locations. Interestingly enough, the comics take place in New York a lot. There would be a lot to work with, and we can compare how they each present Terra's story. Lastly, a map used by a villain in the show put Jump City in California. The wiki of Teen Titans mentioned that Jump City is based on San Francisco. Another solution could be to use an original layout of San Franciso and finesse a map from what we find the descriptions of setting from the XML.

Sources:
http://teentitansinfo.weebly.com/ (a map I found of Jump City, but it's small and hard to read)
https://teen-titans-go.fandom.com/wiki/Jump_City
https://teentitans.fandom.com/wiki/Jump_City

- The second idea mostly uses the texts and is a graph. It's also pretty ridiculous, but I'm still going to present it. We track each character through the season. See, pretty crazy. I'm imagining a line graph with each character having their line that's representative of their color. Now, this might be redundant since they're usually traveling as a team, but I have a reason for considering this graph. I mentioned when proposing teen titans the first time that they're episodes that have more importance for one titan or are focused solely on one titan.  Especially this season, each titan has a different opinion on Terra, and some spend more time with her than others. Also, following them through their route throughout the season could give more individual focus on the characters. Why was Beast Boy there when Robin never appeared there, that type of thing. In the comics, they're mostly traveling alone, so we could compare the comics and the tv series on which route they have the character travel and how it affects them in each. 

- The third idea is the most abstract, and I would love to do it, but it's *sighs*. I've been thinking about what we could do with the popularity, and I was at a loss at first. As I searched Google, I didn't see a lot about the reputation of the Teen Titans. There were a couple of popularity polls, but it was pretty standard as the leading five would always be the top 5 in the survey. However, a brain shutdown made me realize something. Popularity is shown in many different ways. One example is the fact that Beast Boy is one of the weakest in the comics and barely does anything, but in the tv series, arguably has the most content focused around him. The comics were made in 1980 and the tv show in 2003. Any changes made to the storyline were probably made because of the audience, meaning that more people liked Beast Boy when the tv show came out versus the comics when Robin had a fanbase. Beast Boy was a new character that no cared for until after the comics became a hit. SO the idea is to make a network map indicating the shift of the audience. Where did Robin lose to Beast Boy? I don't know how to do this, but I've been obsessing over the idea of showing it. Maybe neither media has anything to do with why, but the development changed for some reason. Not to mention, these are decisions that someone had to make while making the tv show. Since it's based on the comics, they made decisions to change or leave out things out from the comics; why? Dr.B's friend mentioned that it was plausible, so it's got to be. I wish I could show you what's in my mind because I have so many ideas about the third idea.

I think that's all for now! Dr. B did mention that we want to keep in mind of the work we did beforehand, so I tried to make ideas with that in mind. **AGAIN ANY IDEAS DONT BE AFRAID TO MENTION!** My mind is complex. Also, ask questions! I don't have much concrete stuff to work with, so forcing me to think might help make one of my ideas more ideal.

**Note:** If we wanna focus on the visuals instead of text, we could do a graph about the differences of the characters' appearance from the comics to the tv series. Shannon put pictures of the Teen Titans on the site, but there are a lot of characters we could compare.

**Comment by frabbitry** [comment]
As promised, here's a response to these ideas (which are all great, by the way). 

Idea One: This is definitely possible, especially if you can find maps that someone else has created. If you can't, then you guys would have to make a map yourself, which isn't impossible, but can be time-consuming. I also think that this would be one of the easier ideas to implement because it sounds like you guys already have a lot of the data that you need to make it happen, if you're tagging setting information. 

Idea Two: With this idea, you're looking at where each character travels. Sure, you could represent their travels graphically with lines. I have some thoughts on this, though, that I'll spell out later. 

Idea Three: If I kind of understand what you're getting at here, with this idea you'd be trying to look at the point at which certain characters gained popularity over others. In other words (or at least how I see it), you're trying to find the factors that determine popularity for a character. I actually think that this could be a pretty compelling research question to examine because you're right: you would be looking at how the character content shifts in relation to the characters. There are many different ways to measure this content, such as amount of dialogue apportioned to each character and whatnot. This guy named Todd Schneider used R (a pretty cool programming language for statistics and machine learning) [to analyze the first 26 seasons of the Simpsons](https://toddwschneider.com/posts/the-simpsons-by-the-data/), and he analyzed characters and content in a couple of different ways, just as an example. (I think he also posted his code and stuff on GitHub, if you want to see more). Just from that as an example, you could look at how much each character is speaking, where they're speaking, what phrases they're saying, etc. You would definitely know more than I would about what's significant for characters to be doing, saying in Teen Titans than I would, but looking at this might lead to some other interesting questions, such as where are the more popular characters located? What are they doing? Who are they interacting with? And so forth. I know that you guys are really looking to only work with Season 2 this time around (at least that's my impression of what you guys want to do), but further research could involve compiling ALL OF THE data from ALL OF THE seasons and ALL OF THE comics, which could help you to determine where and when exactly popularity shifted from one character to another. If you're also looking at the content surrounding the characters, then you might also be able to come up with an answer for why. 

I don't know if my understanding of idea three is way off from what yours is, @lmcneil7. What do you think? 

I also think I mentioned earlier that you could perhaps combine ideas one and two. With this, you'd be making maps AND putting the characters ON those maps. Instead of showing characters' travels on lines, why not show them on a map? This way, you could display the who is traveling with whom, the places that they are traveling to, and perhaps also the significance of those locations, all at the same time. The characters on a map can be considered in context of their relative locations. 

A different DH project that went on when I was taking this class was the [Pokemon Project](http://pokemon.newtfire.org/index.html), and it did something kinda similar; they created a map and were interested in looking at which Pokemon were on which routes or in which places. I don't think that they were interested in examining the movement or the travel of Pokemon as much, but, still, characters ON maps, especially fictional maps. 

If you're really into looking at how the characters MOVE on maps, then you might even consider animating them. It might be difficult, though, and I don't know yet about how you'd do this with a fictional map. Are border crossings significant? Is it significant that a character moves from one place to another with specific other characters? I'm a research assistant for a different DH project right now that's looking at the movement of female characters in politically segregated Belfast in contemporary Irish women's literature. I don't know if you guys know anything about Troubles-era (70s-90s) Belfast, but it kind of sucked and there was a lot of political segregation and violence between Catholics and Protestants; different neighborhoods and streets were politically coded, and moving between them was a BIG DEAL. Furthermore, part of what we're analyzing is that the movement of these female characters between politically coded areas of Belfast is happening primarily when they're in the company of a male character, so it matters who they're going with. For that project, we're animating the maps, because we're really concerned with the movements of characters across borders, and it's a really neat way of showing how the female characters are moving only with male characters. That was kind of a lengthy diversion but I'm just trying to say that animating your maps is a possibility if you're interested in creating a graphical representation of Season 2 and the comics that displays that kind of information. 

I think that you could also do a series of static maps, though. 

In conclusion, I sincerely apologize for the length of this response. Congratulations if you've made it this far! I get really excited about applying statistical methods to literature and literary mapping, and so I'm just a little bit excited because it looks like your research questions may possibly have a little bit to do with either of those things. I'm of course interested to see what you guys eventually end up going with. Hopefully this long rant may have given you some ideas or may have at least helped you all in brainstorming. Let me know what you guys think, and I'd be happy to keep brainstorming with y'all and talking about this, too.

**Comment by lmcneil7** [comment]
Thanks, Fiona, so much! I tend to go on tangents when it comes to ideas, so this was very helpful to scale each of them. You did mention each was doable and helped to show how with such good advice. I still don't know what direction I want to go in. Feedback is the discussion I wanted and will work with what you said to shape my ideas more.

**Idea 1 and 2:** I think combining is a good idea. I was worried that it might be too much, which is why I separated the two approaches, but, in reality, blending the line graph with the map might give it more depth. "Is it significant that a character moves from one place to another with specific other characters?"; this is a cool way to analyze the movement as a team. As I mentioned, some characters have different opinions about Terra, and we could examine whether or not their different beliefs affect who they travel with, etc. It could also show if certain teen titans go in certain groups more frequently or not. ALSO ALSO, it's one of the reasons I wanted to study their routes. One way I look at directing a scene is that you need to focus on who's involved in the scene and who isn't, naturally, each character's movement. Meaning, the difference between leaving a character out of a scene(location or fight) or having them be there and be out-of-commissioned/ unable to fight/ no lines. Routes are an excellent way to give each Titan they're own individuality, and indicating this alongside the map of Jump City could show how their paths apply to the city. Through a quick overview of season 2, the borders crossing are usually to space. Both Starfire and Beast Boy have focused episodes that lead them to space for similar reasons, to escape the other titans. Their focused events show that both of them felt isolated, and through this deviant in their route from the others, it shows more about how they think (ex. Starfire being an alien, Beast Boy feeling that he bothers the other titans). There's a lot to work with here, so this is good. I did see a map through a quick search but might be able to find a better one with more research.

Now, the animating the characters onto a map! I'm freaking, and that would be awesome! I think Undertale might have some experience with animating characters (ex. the dog on their site), but if we wanted to do it, I think we could teach ourselves. I would say we could leave it open as an option and then navigate away while we do the more significant parts.

**Idea 3:** YOU GOT IT! That's cool; you understand my pudding brain, so this semester is going to be epic. The whole idea is to find reasons why some characters are more popular than others. And guess what, we already have data for the purpose! Our SVG Graphs show who speaks the most in each episode. We could include the third idea into our project by giving it a part in the Analysis page and explaining what it shows about the popularity of each character. I looked at _the Simpsons_ link you provided, and another thing they analyzed for prevalence was locations. If we decide to do the map/ routes, we would also have the information necessary to talk about the demand for each character depending on location. I don't think we'll have much to say about the sites, however, because I'm pretty sure each episode takes place in a different setting, but we could compare it to the first season. 

Most of what I said just now was if we wanted to consider popularity as a secondary idea. It encompasses what I thought because most of our analysis depends on the characters. If we wanted to focus entirely on this idea, I believe they are a couple of ways of doing it. We could do the analysis based on dialogue and location, but our main focus would be to create a network map indicating the different reasons a particular character is winning. I'm going to include a photo of what I mean, but basically, we research what might affect popularity (we could use similar concepts from _the Simpsons_ one) and create a network map based on the frequency. We then calculate and would get which character is probably the most popular. We could make additional graphs to help make the information more accessible. 
![IMG_0592](https://user-images.githubusercontent.com/54643986/73587013-b49fd680-4483-11ea-9547-8bf4ae41ab7d.JPG)

**This is not an accurate representation of any information!** The map is to show you what I was considering for the third idea. We would show the frequency of each Titan in each category, from high to low. The character with the most high frequencies or we could do it by scoring, giving the high frequency a 3 and the low frequency a 1. 

Another way is similar to the first but includes comics. We do a play by play analysis of what each media does to display Terra's story, and we use the comparison to show the most popular character. We would have a similar network map to the one I provided, but our main focus would be where the graphs deviant. It would show us where the change occurred and imply which character is most popular.

**Comment by ebeshero** [comment]
This project is going places, possibly with animation! Thank you for putting so much wonderful thinking into it right now, @lmcneil7 and @frabbitry ! Now let's see it through!
